tests/test_data_validation_before_insert.py

```diff
@@ -17,34 +16,42 @@
 # This test checks if the script validator will throw an exception if there are any rows with invalid data
 def test_validator_on_rows():
     df = pd.read_csv('tests/test_data_validation_before_insert.csv')
-    check_validator(df, False)
+    check_validator(df, False, 'Some rows have invalid data. Check logs for details.')
 
 
 # This test checks if the script validator will throw an exception if any of the necessary columns are missing
 def test_validator_on_columns():
     lst = [['Name0', 'Address0'], ['Name1', 'Address1'], ['Name2', 'Address2'], ['Name3', 'Address3']]
     df = pd.DataFrame(lst, columns=['name', 'address1'])
-    check_validator(df, False)
+    check_validator(df, False, 'The data frame does not contain the following required column(s): ')
+
+
+# This test checks if the script validator will throw an exception if there are duplicated rows in terms of the columns
+# 'name', 'address1', 'city', 'state', and 'zip'
```

Comment:
I think it checks for duplicate in all columns including serviceSummary , so that should also be mentioned in comment here.